### ISSUE TITLE
IGV, can't view whole genome

### ISSUE THREAD

**Issue by shedurkin** [RobertsLab/resources#2039]
I'm trying to visualize one of our E5 genomes (`Apulcra-genome.fa`) in IGV, but I can only view individual chromosomes at a time. When I click the "view whole genome" button I just get directed to viewing the first chromosome, ntLink_7. Anyone have any ideas how to view the entire genome at once?

![image](https://github.com/user-attachments/assets/19143c39-052a-4784-b220-bd45c0db9b9f)
![image](https://github.com/user-attachments/assets/8b7ef80c-2e08-48a7-8fde-cc30caa7915a)

**Comment by sr320** [comment]
you can only do chromosome at a time... 

define what you are trying to do by viewing "genome"?

**Comment by shedurkin** [comment]
For the built-in genomes in IGV (e.g. hg38) there’s an option to view all chromosomes (and the full tracks) side by side (here’s some example photos from the web app version, since I’m walking rn) Just chromosome 1:All chromosomes (“whole genome view”)On Nov 21, 2024, at 11:38 AM, Steven Roberts ***@***.***> wrote:﻿
you can only do chromosome at a time...
define what you are trying to do by viewing "genome"?

**Comment by sr320** [comment]
my comment was based on stand-alone IGV.

but Why do you want to view all chromosomes together? as you can see with human data.

**Comment by shedurkin** [comment]
The same functionality is present in the desktop IGV for the human genome.It would just be a faster/easier way to see clumps of mRNA-miRNA binding, since they’re pretty spread out. Looking at all ~170 regions of the apulchra genome individually is more time consuming.On Nov 21, 2024, at 12:22 PM, Steven Roberts ***@***.***> wrote:﻿
my comment was based on stand-alone IGV.
but Why do you want to view all chromosomes together? as you can see with human data.

**Comment by sr320** [comment]
gotcha - alternatives could be bedtools and plotting those data..  or something like karyoploteR

**Comment by shedurkin** [comment]
Figured it out, IDV just doesn't support viewing that many scaffolds at the same time (max is ~50)
[IGV tech support ticket with answer](https://groups.google.com/g/igv-help/c/xkT_-uRsey0/m/5_C6aKqwAAAJ)

**Comment by shedurkin** [comment]
Ok so I figured out how to view a much bigger chunk of the genome at once! IGV "whole genome view" is limited to displaying ~50 scaffolds, while the Apulchra genome has ~170 scaffolds. 

 I manually deleted all but the first 50 lines of the Apulchra-genome.fa.fai index file and re-uploaded it to IGV with the (unaltered) Apulchra-genome.fa. It looks like IGV just ignores any fasta sequences that are not represented in the fasta index file. This also didn't cause any problems when loading tracks (e.g. my gff of miRNA-mRNA bindng sites). So I can view 50 genome scaffolds at a time!
 
 I brok up the Apulchra-genome.fa index file into 3 chunks, then loaded 3 separate IGV sessions, so I can view ll three chunks of the genome in one go. It's a little janky, but it does seem to work :)
 
![image](https://github.com/user-attachments/assets/0e9cf98b-2a8c-4e04-94cc-23929cd6c0ef)
